addons/core/translations/resources/en-us.yaml

```diff
@@ -155,6 +155,9 @@ host:
     unknown: Unknown
     aws: AWS
     azure: Azure
+  form:
+    provider:
```

Comment:
Let's consolidate on a single `provider` translation (maybe from host catalog?).  It's the same across all resources in the feature.

Reply:
good idea! I will consolidate

Reply:
translation consolidation is done and merged [here](https://github.com/hashicorp/boundary-ui/pull/955)